taxi/forms.py

```diff
@@ -0,0 +1,20 @@
+from django import forms
+from .models import Driver, Car
+
+
+class DriverCreationForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["username", "first_name", "last_name", "license_number"]
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):

```

Comment:
The `DriverLicenseUpdateForm` is missing validation for the `license_number` field. According to the task requirements, the license number should consist of 8 characters, with the first 3 being uppercase letters and the last 5 being digits .